### ISSUE TITLE
Allow template tags to use the v-text directive

### ISSUE THREAD

**Issue by hutchings90** [vuejs/vue#11305]
### Version
2.5.16

### Reproduction link
[https://jsfiddle.net/ey867nzs/2/](https://jsfiddle.net/ey867nzs/2/)

### Steps to reproduce
Create a Vue app and display a string using mustache syntax, `v-text` on a `div` element, and `v-text` on a `template` element.

### What is expected?
The string is displayed by all three methods.

### What is actually happening?
The string is displayed by mustache syntax and `v-text` on a `div` element, but the string is not displayed by the `v-text` on a `template` element.

---
This is a feature request similar to #7726. I created a new issue since the other one is closed, it is 2 years old, and I have reasons for requesting this feature that were not provided in that issue.

A problem with mustache syntax is that the UI could get cluttered with curly braces if there is a delay in rendering or if there is a bug. A problem with using `v-text` on a DOM element is that undesired CSS styles could get applied to the DOM element. Allowing `v-text` to be used on `template` elements would avoid these problems.

**Comment by posva** [comment]
> A problem with mustache syntax is that the UI could get cluttered with curly braces if there is a delay in rendering or if there is a bug.

That shouldn't happen in production anyway.

v-text sets the textContent of an element, it cannot work on template

**Comment by hutchings90** [comment]
What do you mean, "That shouldn't happen in production anyway?"

**Comment by hutchings90** [comment]
> > A problem with mustache syntax is that the UI could get cluttered with curly braces if there is a delay in rendering or if there is a bug.
> 
> That shouldn't happen in production anyway.
> 
> v-text sets the textContent of an element, it cannot work on template

What do you mean, "That shouldn't happen in production anyway?"

**Comment by posva** [comment]
I meant a bug breaking your app shouldn't be a scenario in production. If your are using in html templates, you can hide them with v cloak

**Comment by hutchings90** [comment]
What about a delay in rendering and CSS issues with using a DOM element?

**Comment by hutchings90** [comment]
I guess you're addressing the delay in rendering with v-cloak, but CSS issues with using a DOM element still hasn't been addressed.

Vue.js has been a great reactive JavaScript tool. 2 years ago, at work, we were looking around at front-end web development framework options. Vue.js seemed like the way to go because of its incremental adoptability. I'm glad we chose it. Vue.js' two main competitors are Angular and React. I've used Angular a little bit, and I prefer Vue.js. I have never used React. I don't know if the other two allow text interpolation in template tags or something similar to avoid mustache syntax. If they don't and Vue.js does, then Vue.js will have an edge over them. If they do, then Vue.js will bridge a gap.
